### PR TITLE
Added Cross-Cloud GCP Detection

### PR DESCRIPTION
**Pull request by 4R9UN**
Required items, please complete
   
   Change(s):
   - New Detection Created

   Reason for Change(s):
   - See guidance below

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   - See guidance below

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

# Guidance <- remove section before submitting
-----------------------------------------------------------------------------------------------------------
## Before submitting this PR please ensure that you have read the following sections and filled out the changes, reason for change and testing complete sections:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

   Version updated:
   - Yes
   - Detections/Analytic Rule templates are required to have the version updated

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
   - Yes/No/Need Help

_Note: If updating a detection, you must update the version field._

> Before the submission has been made, please look at running the KQL and Yaml Validation Checks locally.
> https://github.com/Azure/Azure-Sentinel#run-kql-validation-locally

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes/No/Need Help
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

**Review by 4R9UN — COMMENTED** [review]
update the GUID

**Review by 4R9UN — COMMENTED** [review]
Updated Description under 255 char

**Review by 4R9UN — COMMENTED** [review]
Fixing the Severity error

**Review by 4R9UN — COMMENTED** [review]
fix description error

**Review by 4R9UN — COMMENTED** [review]
fixing syntax error

**Review by 4R9UN — COMMENTED** [review]
Updated the Tactics

**Review by 4R9UN — COMMENTED** [review]
Update the Techniques

**Review by 4R9UN — COMMENTED** [review]
fixing error

**Review by 4R9UN — COMMENTED** [review]
fixing error

**Review by 4R9UN — COMMENTED** [review]
fix error

**Review by 4R9UN — COMMENTED** [review]
Slight Change in description

File: Detections/MultipleDataSources/SuspiciousVMInstanceCreationActivity.yaml [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  GCPUserAgent: GCPUserUA
+  GCPVMName: VMName
+  GCPProjectId: ProjectId
+  GCPVMType: VMType
+  CorrelationWith: "GCPAuditLogs"
+alertDetailsOverride:
+  alertDisplayNameFormat: "IP address {{GCPUserIp}} Assocated with {{AlertName}} found in GCP VM creation event by {{GCPUserUPN}}"
+  alertDescriptionFormat: "This detection correlates '{{ProductName}}' Alert IP addresse Entity found in VM instance creation in GCP {{ProjectId}}. It identifies successful compute instance creation, from suspicious IP addresse. By joining these datasets on network entities and IP addresses, it detects unauthorized Initial access attempts across GCP environments."
+  alertSeverityColumnName: AlertSeverity
+  alertDynamicProperties:
```
Comment (petebryan) [inline Detections/MultipleDataSources/SuspiciousVMInstanceCreationActivity.yaml:130]: What do you think about setting the alert severity based on the original Microsoft alert severity?

File: Detections/MultipleDataSources/SuspiciousVMInstanceCreationActivity.yaml [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    - alertProperty: ProviderName
+      value: "Microsoft Security"
+    - alertProperty: ProductName
+      value: "Defeder"
```
Comment (petebryan) [inline Detections/MultipleDataSources/SuspiciousVMInstanceCreationActivity.yaml:134]: Is this meant to be "Defender"?

File: Detections/MultipleDataSources/SuspiciousVMInstanceCreationActivity.yaml [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+    - alertProperty: ProductComponentName
+      value: "Defeder"
```
Comment (petebryan) [inline Detections/MultipleDataSources/SuspiciousVMInstanceCreationActivity.yaml:136]: Is this meant to be "Defender"?

File: Detections/MultipleDataSources/SuspiciousVMInstanceCreationActivity.yaml (hunk 1 above)
Comment (4R9UN) [reply Detections/MultipleDataSources/SuspiciousVMInstanceCreationActivity.yaml:130]: Yes, That Could be varies based on Microsoft Alerts

**Review by 4R9UN — COMMENTED** [review]
Address Comments

File: Detections/MultipleDataSources/SuspiciousVMInstanceCreationActivity.yaml (hunk 2 above)
Comment (4R9UN) [reply Detections/MultipleDataSources/SuspiciousVMInstanceCreationActivity.yaml]: fixed

**Review by petebryan — APPROVED** [review]

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**